packages/rich-text/src/test/apply-format.js

```diff
@@ -61,7 +61,7 @@ describe( 'applyFormat', () => {
 		};
 		const expected = {
 			...record,
```

Comment:
The records in this file aren't "correct" in that they lack `replacements`, but I suppose if it's enough to satisfy the tests, it's not that big an issue.

Reply:
Yeah, it's a bit like reducer tests. You don't need a full valid state tree to test one.